data/campaigns/The_Deceivers_Gambit/scenarios/12_The_Traitor.cfg

```diff
@@ -628,7 +627,7 @@ Cavalrymen are general-purpose mounted infantry, with moderate damage for a mode
 
         [message]
             speaker=Delfador
-            message= _ "The young prince told us how you tried to claim the throne for yourself, Arand. You are mortally wounded — confess now, and your brother may forgive you in death."
+            message= _ "The young prince Eldred told us how you tried to claim the throne for yourself, Arand. You are mortally wounded — confess now, and King Garard may forgive you in death."
```

Comment:
I also missed this one: Prince Eldred should probably be capitalised. At least it's 'young Prince' for Konrad and Haldric in HttT and TRoW respectively.